src/database/migrations/20220920054800-create-launches.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("launches", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      site_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
```

Comment:
I think we might need to add a `references` key for both the `user_id` and `site_id` to indicate the foreign key reference to the users and sites table.

Reply:
fixed: 6721422